packages/one-app-runner/__tests__/bin/one-app-runner.spec.js

```diff
@@ -264,7 +264,7 @@ test('--modules option is required if --parrot-middleware option is given', () =
   process.argv = ['', '', '--parrot-middleware', '../path/to/dev.middleware.js', '--root-module-name', 'frank-lloyd-root', '--module-map-url', 'https://example.com/module-map.json', '--docker-image', 'one-app:5.0.0'];
   jest.mock('../../src/startApp', () => jest.fn());
   require('../../bin/one-app-runner');
-  expect(consoleErrorSpy.mock.calls).toMatchSnapshot();
+  expect(consoleErrorSpy.mock.calls.toString()).toContain('parrot-middleware -> modules');
```

Comment:
we could look for the 2 call to `console.error` `consoleErrorSpy.mock.calls[2]` if we wanted to be even more specific

Reply:
Updated (7a4cca8)